code/modules/mob/living/carbon/human/human.dm

```diff
@@ -2063,3 +2065,42 @@
 		var/obj/item/organ/internal/eyes/night/N = E
 		if(N.night_vision )
 			N.disable_night_vision()
+
+/mob/living/carbon/human/get_base_systolic()
```

Comment:
alright, instead of using procs here, just use `species.var` directly, since you won't ever have a case where there's a human mob without a species

Reply:
calling procs has a higher overhead than just checking a var on a datum